javascript/entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
I like to use `#!/usr/bin/env` for flexibility.

```suggestion
#!/usr/bin/env sh
```

Reply:
How does that work?

Reply:
you can manipulate the environment if you want a different $SHELL instead of `sh`, for example.

Another good example, when usine `#!/usr/bin/env python` and you have different versions of python, you can manipulate the environment to define which version you want to use.

And it stays more portable between UNIXes (macOS, BSD, ...).

In container environments, we have more control, but keeping this practice is good anyway.